pages/feed/index.tsx

```diff
@@ -126,13 +126,10 @@ const Feeds: WithPageLayout<HighlightSSRProps> = (props: HighlightSSRProps) => {
         image={ogImage}
         twitterCard="summary_large_image"
       />
-      <div
-        className="container flex flex-col gap-16 px-2 pt-12 mx-auto md:px-16 lg:justify-end md:flex-row"
-        ref={topRef}
-      >
-        <div className="flex-col flex-1 hidden gap-6 mt-12 md:flex">
+      <div className="w-full gap-[2rem] justify-center flex flex-col md:gap-6 xl:gap-16 pt-12 md:flex-row" ref={topRef}>
```

Comment:
```suggestion
      <div className="w-full gap-[2rem] justify-center flex flex-col pt-12 md:flex-row" ref={topRef}>
```

The responsive/prefixed `gap` classes are not affecting anything at those breakpoints (i.e. `xl` and `md`), because the flex items the points are already reduced leaving the `gap` with nothing to space out.

So we can remove them.